doc: Added nRF54H20 migration guides for NCS 2.7 (#15760)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/migration/migration_guide_nRF54H20_cs_to_2_7.rst

.. _migration_nrf4h20_to_2.7:

> Migration notes for |NCS| v2.7 for nRF54H20 users
> #################################################

Reviewer: ```suggestion
Migration notes for |NCS| v2.7.0 for nRF54H20 DK users
######################################################
```

> This document describes the new features and changes implemented in |NCS| to take into consideration when migrating your application for the nRF54H20 DK to |NCS| v2.7.

Reviewer: ```suggestion
This document describes the new features and changes implemented in |NCS| to take into consideration when migrating your application for the nRF54H20 DK to |NCS| v2.7.0.
```

> To ensure the nRF54H20 DK runs its components correctly, verify its compatibility with the |NCS| v2.7 release checking the version number on your DK's sticker.

Reviewer: ```suggestion
To ensure the nRF54H20 DK runs its components correctly, verify its compatibility with the |NCS| v2.7.0 release by checking the version number on your DK's sticker.
```

> The only versions of the nRF54H20 DK compatible with the |NCS| v2.7 are the following ones:

Reviewer: ```suggestion
The only versions of the nRF54H20 DK compatible with the |NCS| v2.70 are the following:
```
